test/dynamo/test_functions.py

```diff
@@ -2427,6 +2427,29 @@ def f(x):
         python_out = torch.mm(torch.ones(4, 4, device="cuda"), x) + 10
         self.assertEqual(torch_out, python_out)
 
+    @requires_cuda()
+    def test_triton_kernel_strided_input(self):
+        def f(inp):
+            left, right = torch.split(inp, [128, 128], dim=1)
```

Comment:
Do you mind also adding a test where `right` gets used as the input to the triton kernel? That will exercise a version where we're passing a tensor with a non-zero storage offset to triton.

I think... we shouldn't need to to preserve the storage_offset when we clone, but it would be good to have a test for it.

Reply:
Sure, will do!